src/libraries/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/IFormatter.cs

```diff
@@ -8,9 +8,11 @@ namespace System.Runtime.Serialization
 {
     public interface IFormatter
     {
+        internal const string RequiresDynamicCodeMessage = "BinaryFormatter serialization uses dynamic code generation, the Type of objects being processed cannot be statically discovered.";
```

Comment:
```suggestion
        internal const string RequiresDynamicCodeMessage = "BinaryFormatter serialization uses dynamic code generation, the type of objects being processed cannot be statically discovered.";
```
Nit: The capitalized `Type` looks odd to me. This is the only RUC or RDC message that uses capitalized `Type` in the repo. I know that you are mirroring the existing message. If you choose to fix this, could you please fix the existing message as well.